cloudinary_cli/modules/sync.py

```diff
@@ -395,8 +414,11 @@ def _handle_unique_local_files(self):
         logger.info(f"Deleting {len(self.unique_local_file_names)} local files...")
         for file in self.unique_local_file_names:
             full_path = path.abspath(self.local_files[file]['path'])
-            remove(full_path)
-            logger.info(f"Deleted '{full_path}'")
+            if not self.dry_run:
```

Comment:
Pretty similar to the previous comment, it's much easier to read code when the main function logic is not under condition.
If you could rewrite it like:
```
if self.dry_run:
    logger.info(f"Dry run mode enabled. Would delete '{full_path}'")
    continue

<ACTUAL LOGIC GOES HERE>
```

it would be great!